src/EnergyPlus/DataHVACGlobals.hh

```diff
@@ -75,6 +75,7 @@ namespace DataHVACGlobals {
 
     // MODULE PARAMETER DEFINITIONS:
 
+    Real64 const SmallHumRatDiff(1.0E-7);
```

Comment:
Using new approach for global constants - just declare it right in the header. Better for compiler optimization instead of using `extern`. Tempted to change all of these here, but don't want to muddy the waters.